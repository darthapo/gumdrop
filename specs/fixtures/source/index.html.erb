<h1>Tests!</h1>
<ul>
<% site.contents.keys.sort.each do |uri| next if site.in_blacklist?(uri) %>
  <li><a href="<%= uri %>"><%= uri %></a></li>
<% end %>
</ul>