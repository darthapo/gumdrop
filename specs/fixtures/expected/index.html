<body class="site"><h1>Tests!</h1>
<ul>
  <li><a href="data-access/from-csv.html">data-access/from-csv.html</a></li>
  <li><a href="data-access/from-json.html">data-access/from-json.html</a></li>
  <li><a href="data-access/from-sqlite.html">data-access/from-sqlite.html</a></li>
  <li><a href="data-access/from-xml.html">data-access/from-xml.html</a></li>
  <li><a href="data-access/from-yaml.html">data-access/from-yaml.html</a></li>
  <li><a href="data-access/from-yamldb.html">data-access/from-yamldb.html</a></li>
  <li><a href="data-access/from-yamldoc.html">data-access/from-yamldoc.html</a></li>
  <li><a href="gen-with-block/as-file.html">gen-with-block/as-file.html</a></li>
  <li><a href="gen-with-block/as-plain.html">gen-with-block/as-plain.html</a></li>
  <li><a href="gen-with-block/layout-nested.html">gen-with-block/layout-nested.html</a></li>
  <li><a href="gen-with-block/layout-nil.html">gen-with-block/layout-nil.html</a></li>
  <li><a href="gen-with-block/layout-wrap.html">gen-with-block/layout-wrap.html</a></li>
  <li><a href="gen-with-inline-render/0-as-file.html">gen-with-inline-render/0-as-file.html</a></li>
  <li><a href="gen-with-inline-render/0-as-plain.html">gen-with-inline-render/0-as-plain.html</a></li>
  <li><a href="gen-with-inline-render/0-layout-nil.html">gen-with-inline-render/0-layout-nil.html</a></li>
  <li><a href="gen-with-inline-render/0-layout-sub.html">gen-with-inline-render/0-layout-sub.html</a></li>
  <li><a href="gen-with-inline-render/0-layout-wrap.html">gen-with-inline-render/0-layout-wrap.html</a></li>
  <li><a href="gen-with-inline-render/1-as-file.html">gen-with-inline-render/1-as-file.html</a></li>
  <li><a href="gen-with-inline-render/1-as-plain.html">gen-with-inline-render/1-as-plain.html</a></li>
  <li><a href="gen-with-inline-render/1-layout-nil.html">gen-with-inline-render/1-layout-nil.html</a></li>
  <li><a href="gen-with-inline-render/1-layout-sub.html">gen-with-inline-render/1-layout-sub.html</a></li>
  <li><a href="gen-with-inline-render/1-layout-wrap.html">gen-with-inline-render/1-layout-wrap.html</a></li>
  <li><a href="goodbye.html">goodbye.html</a></li>
  <li><a href="hello.html">hello.html</a></li>
  <li><a href="image.png">image.png</a></li>
  <li><a href="index.html">index.html</a></li>
  <li><a href="js/sprockets-app.js">js/sprockets-app.js</a></li>
  <li><a href="js/stitch-app.js">js/stitch-app.js</a></li>
  <li><a href="js/stitch-app.min.js">js/stitch-app.min.js</a></li>
  <li><a href="js/stitch-app.min.src.js">js/stitch-app.min.src.js</a></li>
  <li><a href="js/straight.js">js/straight.js</a></li>
  <li><a href="js/test-coffee.js">js/test-coffee.js</a></li>
  <li><a href="pages/docs/force-abs.html">pages/docs/force-abs.html</a></li>
  <li><a href="pages/docs/index.html">pages/docs/index.html</a></li>
  <li><a href="partials/cached.html">partials/cached.html</a></li>
  <li><a href="partials/hoisted-data.html">partials/hoisted-data.html</a></li>
  <li><a href="partials/nested.html">partials/nested.html</a></li>
  <li><a href="partials/params.html">partials/params.html</a></li>
  <li><a href="partials/with-layouts.html">partials/with-layouts.html</a></li>
  <li><a href="robots.txt">robots.txt</a></li>
  <li><a href="test-multi-procs.html">test-multi-procs.html</a></li>
  <li><a href="test-nested-layout.html">test-nested-layout.html</a></li>
  <li><a href="test-no-layout.html">test-no-layout.html</a></li>
  <li><a href="test-yamldoc.html">test-yamldoc.html</a></li>
  <li><a href="test.html">test.html</a></li>
</ul></body>